.drone.jsonnet

```diff
@@ -11,17 +11,27 @@ local server_ref_map = {
 };
 
 local builddir = 'verylongdirnameforverystrangecpackbehavior';
-local cmakeflags = '-DCMAKE_BUILD_TYPE=RelWithDebInfo -DPLUGIN_COLUMNSTORE=YES -DPLUGIN_XPAND=NO -DPLUGIN_MROONGA=NO -DPLUGIN_ROCKSDB=NO -DPLUGIN_TOKUDB=NO -DPLUGIN_CONNECT=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_SPHINX=NO';
+
+// This is stupid use of PLUGIN_*=NO syntax. It should just skip building
+// _everything_ else apart from the things that ColumnStore actually needs.
+local cmakeflags = '-DCMAKE_BUILD_TYPE=RelWithDebInfo -DPLUGIN_COLUMNSTORE=YES -DPLUGIN_XPAND=NO -DPLUGIN_MROONGA=NO -DPLUGIN_ROCKSDB=NO ' +
+                   '-DPLUGIN_TOKUDB=NO -DPLUGIN_CONNECT=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_SPHINX=NO ' +
+                   '-DPLUGIN_PERFSCHEMA=NO -DWITH_EMBEDDED_SERVER=OFF -DWITH_WSREP=OFF';
```

Comment:
IIRC our bulk insertion buffer depends on perfschema so we can't compler w/o it. I believe DWITH_WSREP disables async replication but we do use this ocassionaly in the tests for dimension innodb tables.

Reply:
Apparently you can compile and the test suite passes as the CI shows? Or is the CI unreliable, it does not actually run a ColumnStore test suite?

Reply:
Yes, ATM the feature is in testing so production MTR that is also under development doesn't have tests for the feature.

Reply:
I've added perfschema back in PR#1702 for develop-5, which you should merge, since you seem to merge from develop-5 onwards to develop and this PR will become obsolete.